templates/web/components/head.tmpl

```diff
@@ -29,6 +29,8 @@
 		<link rel="icon" type="image/png" sizes="96x96" href='{{ static_path "bmvii_favicon-96x96.png" }}'>
 		{{ end }}
 
+        <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Cabin" />
```

Comment:
```suggestion
        <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Cabin" />
```
Adding https seems to fix this in firefox

Reply:
Looks like it works on safari as well

![Screen Shot 2020-10-24 at 11 18 18 PM](https://user-images.githubusercontent.com/31838229/97098073-46252900-164f-11eb-8e38-45c3c442635c.png)